projects/bp-gallery/src/components/common/PictureOverview.tsx

```diff
@@ -32,10 +34,14 @@ const PictureOverview = ({
     false,
     sortBy,
     true,
-    ABSOLUTE_MAX_PICTURES_PER_ROW * rows
+    ABSOLUTE_MAX_PICTURES_PER_ROW * rows,
+    'cache-and-network',
+    type
   );
 
-  const pictures: FlatPicture[] | undefined = useSimplifiedQueryResponseData(data)?.pictures;
+  const pictures: FlatPicture[] | undefined = useSimplifiedQueryResponseData(
+    data as { [key: string]: any } | undefined
```

Comment:
shouldn't be needed here, data already interferes a much better type from the getPictures query